src/com/example/ppj07/Task1.java

```diff
@@ -0,0 +1,37 @@
+package com.example.ppj07;
+
+import java.util.Scanner;
+
+public class Task1 {
+    public static void main(String[] args) {
+        task1V1();
+        task1V2();
+    }
+
+    public static void task1V1() {
+        Scanner scanner = new Scanner(System.in);
+        int a = scanner.nextInt();
+        int b = scanner.nextInt();
+        int c = scanner.nextInt();
+        if (c < a && c < b) {
+            System.out.println(a + " " + b);
+        } else if (b < a && b < c) {
+            System.out.println(a + " " + c);
+        } else {
+            System.out.println(b + " " + c);
+        }
+    }
+
+    public static void task1V2() {
+        Scanner scanner = new Scanner(System.in);
+        int max1 = scanner.nextInt();
+        int max2 = scanner.nextInt();
+        int temp = scanner.nextInt();
+        if (temp >= max1) {
+            max1 = temp;
+        } else if (temp >= max2) {
+            max2 = temp;
+        }
```

Comment:
1. The task is to read 5 numbers and find 3 greatest values. In your implementation only 3 values read.
2. In this implementation that finds two greatest values, max1 and max2 calculated not correct for all cases, example for the next values:
```
   int max1 = scanner.nextInt(); // = 2
   int max2 = scanner.nextInt(); // = 1
   int temp = scanner.nextInt(); // = 3
```
the result will be "3 1", but should be "3 2"